KimYeSeung/210930/Scripts/Gamemanager.cs

```diff
@@ -0,0 +1,144 @@
+using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+
+public class Gamemanager : MonoBehaviour
+{
+    public int totalPoint;
+    public int stagePoint;
+    public int stageIndex;
+    public int health;
+    public int HpPoint;
+
+    public playermove player;
+    public Image[] UIHp;
+    public Text UIPoint;
+    public GameObject UIRestartBtn;
+
+    public GameObject UIContinueBtn;
+    public GameObject UIEndBtn;
+    public GameObject Menu;
+    public GameObject Tutorial;
+    public GameObject MainStage;
+
+
+    void Awake(){
+        health =3;
+        UIHp[3].color = new Color(0,0,0,0);
+        UIHp[4].color = new Color(0,0,0,0);
+        UIHp[5].color = new Color(0,0,0,0);
+        UIHp[6].color = new Color(0,0,0,0);
+        health=HpPoint;
+    }
+
+    void Update() {
+        //점수 표시
+        UIPoint.text = (totalPoint + stagePoint).ToString();
```

Comment:
```suggestion
        UIPoint.text = $"{totalPoint + stagePoint}";
```
보간 문자열을 아십니까? 위와 같이 처리할 수 있습니다.

Reply:
`ToString` 메소드는 박싱을 일으키기 때문에 여기서 부하를 일으킬 수 있습니다. 박싱 언박싱에 대한 개념은 직접 구글링해서 찾아보고 꼭 이해하길 바라는 개념 중 하나입니다.